controllers/openshift_controller.go

```diff
@@ -210,14 +190,6 @@ WantedBy=multi-user.target
 			APIVersion: "machineconfiguration.openshift.io/v1",
 			Kind:       "MachineConfig",
 		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "50-enable-sandboxed-containers-extension",
-			Labels: map[string]string{
-				"machineconfiguration.openshift.io/role": machinePool,
-				"app":                                    r.kataConfig.Name,
-			},
-			Namespace: "sandboxed-containers-operator",
-		},
```

Comment:
Wasn't this part that I mistakenly removed?  Shouldn't we also backport the patch re-adding it?

Reply:
no that was the name of the machine config that we added back, not the namespace

Reply:
Perfect then!

Reply:
now I see this removes indeed the entire thing not just the namespace. so you were right and we need to backport my fix as well.

Reply:
ack, I'm removing the lgtm for now and will re-add as soon as this one is updated. Thanks!